### PR TITLE
Removing unused code from TradingAccount.py

### PR DESCRIPTION
**Pull request by edrickrenan**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please make your selection.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a new dependency
- [x] Code Maintainability / comments
- [ ] Code Refactoring / future-proofing

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added pytest unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: models/TradingAccount.py [hunk 1]
```diff
@@ -289,345 +294,45 @@ def saveTrackerCSV(self, market='', save_file='tracker.csv'):
...
+                                           np.add(df_tracker['buy_fees'], df_tracker['sell_fees']))
         df_tracker['margin'] = np.multiply(np.true_divide(df_tracker['profit'], df_tracker['buy_value']), 100)
         df_sincebot = df_tracker[df_tracker['buy_at'] > '2021-02-1']
 
         try:
             df_sincebot.to_csv(save_file, index=False)
         except OSError:
-            raise SystemExit('Unable to save: ', save_file) 
-
-    def buy(self, cryptoMarket, fiatMarket, fiatAmount=0, manualPrice=0.00000000):
```
Comment (edrickrenan) [inline models/TradingAccount.py:334]: The buy method used on pycryptobot is `PyCryptoBot::marketBuy`

File: models/TradingAccount.py [hunk 2]
```diff
@@ -289,345 +294,45 @@ def saveTrackerCSV(self, market='', save_file='tracker.csv'):
...
-
-                # update the dummy fiat balance
-                self.balance.loc[self.balance['currency'] == fiatMarket, 'balance'] = self.getBalance(fiatMarket) - fiatAmount
-                self.balance.loc[self.balance['currency'] == fiatMarket, 'available'] = self.getBalance(fiatMarket) - fiatAmount
-
-                # update the dummy crypto balance
-                self.balance.loc[self.balance['currency'] == cryptoMarket, 'balance'] = self.getBalance(cryptoMarket) + (fiatAmountMinusFee / price)
-                self.balance.loc[self.balance['currency'] == cryptoMarket, 'available'] = self.getBalance(cryptoMarket) + (fiatAmountMinusFee / price)
-
-    def sell(self, cryptoMarket, fiatMarket, cryptoAmount, manualPrice=0.00000000):
```
Comment (edrickrenan) [inline models/TradingAccount.py:488]: The sellmethod used on pycryptobot is `PyCryptoBot::marketSell`

**Comment by whittlem** [comment]
These huge PR's make me nervous now. Far too many changes all at once and makes testing the change really difficult. I would prefer smaller PR's that make it easier to review.

**Comment by edrickrenan** [comment]
The only change here is to remove 2 unused methods.

**Comment by dthevenin** [comment]
> The only change here is to remove 2 unused methods.

Maybe next time we could separate the styling fix (that is huge but its only styling) from the real change, and have 2 commits.

File: models/TradingAccount.py [hunk 3]
```diff
@@ -103,11 +98,12 @@ def getOrders(self, market='', action='', status='all'):
                 if market == '':
                     return self.orders
                 else:
-                    return self.orders[self.orders['market'] == market]             
+                    return self.orders[self.orders['market'] == market]
         if self.app.getExchange() == 'coinbasepro':
             if self.mode == 'live':
                 # if config is provided and live connect to Coinbase Pro account portfolio
-                model = CBAuthAPI(self.app.getAPIKey(), self.app.getAPISecret(), self.app.getAPIPassphrase(), self.app.getAPIURL())
+                model = CBAuthAPI(self.app.getAPIKey(), self.app.getAPISecret(), self.app.getAPIPassphrase(),
```
Comment (dthevenin) [inline models/TradingAccount.py:105]: its usually bette to have either, everything in the same line, or one parameter per line.
I guess here we should have
```python
      CBAuthAPI(
        self.app.getAPIKey(),
        self.app.getAPISecret(),
        self.app.getAPIPassphrase(),
        self.app.getAPIURL()
      )
```

File: models/TradingAccount.py (hunk 1 above)
Comment (dthevenin) [reply models/TradingAccount.py:334]: @whittlem Do you remember why those duplicate functions?
I was thinking to remove them when I did the big refactoring, but it was already too much...

**Review by dthevenin — COMMENTED** [review]
I did not test yet the change looks good.
I don't want to be picky with style, so up to you, just usually it's better to put one parameter per line.
I will test in case.

**Review by dthevenin — APPROVED** [review]
I did run the bot and it seems to work well.
Unfortunately the tests are failing for me but its not related to that PR, since there failing the same for master.
It LGTM but it depends on why these functions were there at the first place I think.

**Review by dthevenin — APPROVED** [review]
LGTM

**Comment by marioK93** [comment]
the change looks safe, but i think we should try understand what these functions were originally intended for as the api used seems to be a sandbox for coinbase

@whittlem do you remember what the coinbase sandbox was intended for?

**Review by whittlem — APPROVED** [review]
Thanks for cleaning this up.

**Pull request merged**